Projects/App/Sources/Register/GiftionImageViewController.swift

```diff
@@ -30,6 +30,8 @@ final class GiftionImageViewController: UIViewController {
     }
     
     private func configure() {
+        view.backgroundColor = .white
```

Comment:
```suggestion
        view.backgroundColor = .ddip(.neutralWhite))
```

designSystem을 사용해주세요 😅

Reply:
흰색의 경우 모두 designSystem을 사용하도록 변경했습니다~!